ReactiveCocoa/Swift/Bag.swift

```diff
@@ -60,7 +60,7 @@ internal struct Bag<T> {
 	/// will reset all current identifiers to reclaim a contiguous set of
 	/// available identifiers for the future.
 	private mutating func reindex() {
-		for var i = 0; i < elements.count; i++ {
+		for i in 0..<elements.count {
```

Comment:
@NachoSoto Thanks for the tip! I swapped out count with endIndex. 
For the startIndex I think leaving that as 0 is fine but endIndex is much better than what I had before with using count.